solutions/basic/stack_definition.json

```diff
@@ -617,9 +650,17 @@
           "name": "prefix",
           "value": "ref:../../inputs/prefix"
         },
+        {
+          "name": "create_icr_namespace",
+          "value": true
+        },
         {
           "name": "add_container_name_suffix",
           "value": true
+        },
+        {
+          "name": "add_code_engine_prefix",
+          "value": false
```

Comment:
Is there a reason this is `false`? @in-1911 any preference to what this should be?

Reply:
Yes, we already have the prefix added. This is to prevent ALM adding it a second time.